projects/demo/src/modules/components/tag/examples/4/index.ts

```diff
@@ -0,0 +1,22 @@
+import {Component} from '@angular/core';
+import {changeDetection} from '@demo/emulate/change-detection';
+import {encapsulation} from '@demo/emulate/encapsulation';
+
+@Component({
+    selector: 'tui-tag-example-4',
+    templateUrl: './index.html',
+    styleUrls: ['./index.less'],
+    changeDetection,
+    encapsulation,
+})
+export class TuiTagExample4 {
+    readonly tags = ['Taiga UI', 'is an open-source library', 'for awesome people'];
+
+    handleTagEdited(newTagCaption: string, tagIndex: number) {
+        if (newTagCaption) {
+            this.tags[tagIndex] = newTagCaption;
```

Comment:
nit: mutating elements by index is bad practice

Reply:
Ok, also done it with splice 👍